helm/jaeger-operator-app/templates/collector-service.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.collector.service.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "jaeger-operator.fullname" . }}-admin
+  labels:
+{{ include "jaeger-operator.labels" . | indent 4 }} 
+spec:
+  ports:
+  - name: admin-http
```

Comment:
I think the convention is to the metrics port "metrics" but I actually don't understand prometheus serviceMonitors.

Reply:
"metrics" is already in use by operator as `jaeger-jaeger-operator-app-metrics` servicemonitor and service are deployed on our cluster. and is used for a very minimal set of metrics mainly for operator it self as far as I remember. but this about exposing the `admin-http` port defined in deployment resource for Jaeger. ie.:

```
- containerPort: 14269
  name: admin-http
  protocol: TCP
```